### PR TITLE
Initialize training from frozen model

### PR DESCRIPTION
**Pull request by reuben**
This flag allows one to resume training from a frozen model, but as documented on the flag, it does not restore optimizer parameters, so adjusting the learning rate is necessary. I think it's an OK requirement from users.

### CODE REVIEW

File: DeepSpeech.py [hunk 1]
```diff
@@ -1560,6 +1584,12 @@ def end(self, session):
                                                checkpoint_dir=FLAGS.checkpoint_dir,
                                                save_checkpoint_secs=FLAGS.checkpoint_secs if FLAGS.train else None,
                                                config=session_config) as session:
+            if len(FLAGS.initialize_from_frozen_model) > 0:
+                log_info('Initializing from frozen model: {}'.format(FLAGS.initialize_from_frozen_model))
+                feed_dict = {}
+                model_feeder.set_data_set(feed_dict, model_feeder.train)
```
Comment (kdavis-mozilla) [inline DeepSpeech.py]: Kind of odd that this has to be done as the run below shouldn't need any of the data, but ¯\_(ツ)_/¯
